cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java

```diff
@@ -451,4 +459,59 @@ private void setProducerUsernameAndPassword(String username, PasswordSecretSourc
     protected String getWhitelist() {
         return whitelist;
     }
+
+    public ServiceAccount generateInitContainerServiceAccount() {
+        return new ServiceAccountBuilder()
+                .withNewMetadata()
+                .withName(initContainerServiceAccountName(cluster))
+                .withNamespace(namespace)
+                .withOwnerReferences(createOwnerReference())
+                .withLabels(labels.toMap())
+                .endMetadata()
+                .build();
+    }
+
+    /**
+     * Get the name of the mirror maker service account given the name of the {@code mirrorMakerResourceName}.
+     */
+    public static String initContainerServiceAccountName(String mirrorMakerResourceName) {
+        return kafkaMirrorMakerClusterName(mirrorMakerResourceName);
+    }
+
+    /**
+     * Get the name of the mirror maker init container role binding given the name of the {@code namespace} and {@code cluster}.
+     */
+    public static String initContainerClusterRoleBindingName(String namespace, String cluster) {
+        return "strimzi-" + namespace + "-" + cluster + "-mirror-maker-init";
+    }
+
+    /**
+     * Creates the ClusterRoleBinding which is used to bind the MirrorMaker SA to the ClusterRole
+     * which permissions the MirrorMaker init container to access K8S nodes.
+     */
+    public KubernetesClusterRoleBinding generateClusterRoleBinding(String assemblyNamespace) {
+        KubernetesSubject ks = new KubernetesSubjectBuilder()
+                .withKind("ServiceAccount")
+                .withName(initContainerServiceAccountName(cluster))
+                .withNamespace(assemblyNamespace)
+                .build();
+
+        KubernetesRoleRef roleRef = new KubernetesRoleRefBuilder()
+                .withName("strimzi-kafka-mirror-maker-broker")
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("ClusterRole")
+                .build();
+
+        return new KubernetesClusterRoleBindingBuilder()
+                .withNewMetadata()
+                .withName(initContainerClusterRoleBindingName(namespace, cluster))
+                .withNamespace(assemblyNamespace)
+                .withOwnerReferences(createOwnerReference())
+                .withLabels(labels.toMap())
+                .endMetadata()
```

Comment:
As a general comment please indent the code between `withNew*()` and `end*()`, that way the indentation reflects the nesting structure of the objects being built and makes the code much easier to read.